dbms/src/Storages/DeltaMerge/File/DMFilePackFilter.h

```diff
@@ -68,9 +68,10 @@ class DMFilePackFilter
         return pack_filter;
     }
 
-    inline const std::vector<RSResult> & getHandleRes() const { return handle_res; }
-    inline const std::vector<UInt8> & getUsePacksConst() const { return use_packs; }
-    inline std::vector<UInt8> & getUsePacks() { return use_packs; }
+    const RSResults & getHandleRes() const { return handle_res; }
+    const RSResults & getPackResConst() const { return pack_res; }
```

Comment:
maybe we can combine `handle_res` and `pack_res`?

Reply:
> `pack_res` is the filter results of `rowkey_ranges && filter && read_packs`.

Add this comment to line 171